contracts/instruments/RibbonThetaVault.sol

```diff
@@ -147,12 +150,25 @@ contract RibbonThetaVault is DSMath, OptionsVaultStorage {
         __Ownable_init();
         transferOwnership(_owner);
         cap = _initCap;
+        isSunset = false;
+        replacementVault = address(0);
```

Comment:
We can avoid setting these values because they default to `false` and `address(0)` anyway